src/repl/scala/tools/nsc/interpreter/IMain.scala

```diff
@@ -970,7 +970,7 @@ class IMain(initialSettings: Settings, protected val out: JPrintWriter) extends
     // the type symbol of the owner of the member that supplies the result value
     lazy val resultSymbol = {
       val sym = lineRep.resolvePathToSymbol(fullAccessPath)
-      // plow through the INSTANCE member when -Yrepl-class-based
+      // plough through the INSTANCE member when -Yrepl-class-based
```

Comment:
Since it was my contribution, I'd say gratuitous spelling changes are a step below whitespace. The code base hasn't settled on `behavio(u)r` yet. Or actually, there's `AnsiColor`. That's not a great example because the constant names are nonstandard uppercase. I could get behind a spelling change along the lines of `io.Source` -> `io.Sores`.